pkg/migration/runner.go

```diff
@@ -599,6 +607,29 @@ func (r *Runner) createCheckpointTable(ctx context.Context) error {
 	return nil
 }
 
+func (r *Runner) GetProgress() Progress {
+	var summary string
+	switch r.getCurrentState() { //nolint: exhaustive
```

Comment:
Add a default to check the inevitable future bug where, somehow, it's none of the proper cases. That way, user doesn't get an empty string; they can at least get something slightly better like "Error: status unknown (report this as a bug )".